### ISSUE TITLE
Refreshing any reports pages leads to the displaying an empty state

### ISSUE THREAD

**Issue by abarghoud** [bitwarden/clients#8560]
### Steps To Reproduce

1. Go to reports page '#/reports'
2. Click on any report page, for example 'Reused passwords'
3. Results appears
4. Refresh the page

### Expected Result

Shows the report's results

### Actual Result

'Good news' empty state appears

### Screenshots or Videos

<img width="930" alt="image" src="https://github.com/bitwarden/clients/assets/13522017/61bb3436-bb39-4232-ad71-b4ecf9864a41">

### Operating System

macOS

### Web Browser

Chrome

### Build Version

Current production version

### Issue Tracking Info

- [X] I understand that work is tracked outside of Github. A PR will be linked to this issue should one be opened to address it, but Bitwarden doesn't use fields like "assigned", "milestone", or "project" to track progress.

**Comment by Krychaz** [comment]
Hi there,

Thank you for your report! 

I was able to reproduce this issue, and I have flagged this to our engineering team.

If you wish to add any further information/screenshots/recordings etc., please feel free to do so at any time - our engineering team will be happy to review these.

Thanks once again!

**Comment by abarghoud** [comment]
@Krychaz Can I try to work on it, or does it have to be resolved by your engineering team?

**Comment by Krychaz** [comment]
Absolutely! You can find our contribute docs here - https://contributing.bitwarden.com/
